@nav-frontend/accordion-styles/src/index.css

```diff
@@ -0,0 +1,81 @@
+.ReactCollapse--collapse {
+  transition: height 200ms ease-out;
+}
+.navds-accordion {
+  position: relative;
+  background: var(--navds-color-background);
+  border-radius: 0.25rem;
+}
+
+.navds-accordion:hover {
+  box-shadow: var(--navds-shadow-hover);
+  border: var(--navds-panel-hover);
+}
+
+.navds-accordion {
```

Comment:
To `.navds-accordion`-regler. Rart linteren ikke reagerer på det. Er den feil confet, kanskje?

Reply:
Fikset, regner med det er valid siden ingen regler ble overskrevet kanskje?